schemas/BaseElements.xsd

```diff
@@ -264,12 +264,12 @@
 		<xs:complexContent>
 			<xs:extension base="InsulationLayerInfoBase">
 				<xs:sequence>
-					<xs:element name="DistanceToFoundationWallTop" type="LengthMeasurement" minOccurs="0">
+					<xs:element name="DistanceToTopOfInsulation" type="LengthMeasurement" minOccurs="0">
 						<xs:annotation>
 							<xs:documentation>[ft] Vertical distance from top of insulation layer to top of foundation wall.</xs:documentation>
 						</xs:annotation>
 					</xs:element>
-					<xs:element name="InsulationLayerHeight" type="LengthMeasurement" minOccurs="0">
+					<xs:element name="DistanceToBottomOfInsulation" type="LengthMeasurement" minOccurs="0">
 						<xs:annotation>
 							<xs:documentation>[ft] Height of vertical insulation layer.</xs:documentation>
```

Comment:
@yzhou601 Looks like this documentation needs to be updated too?

Reply:
Yes I addressed it at the last commit